CHANGELOG.md

```diff
@@ -113,10 +141,6 @@ These should not affect the functionality of the module.
 
 [Full Changelog](https://github.com/voxpupuli/puppet-openvpn/compare/v7.0.0...v7.1.0)
 
-**Implemented enhancements:**
-
-- add openvpn::deploy::\(export/client\) [\#261](https://github.com/voxpupuli/puppet-openvpn/pull/261) ([to-kn](https://github.com/to-kn))
```

Comment:
Can you explain why this moved?

Reply:
I did not found why.

Reply:
It'd be nice to find out exactly which commit broke it and why and open an issue.  Meanwhile... do you just want to use a version that works or manually fix up this PR?

Reply:
The moved item is not moved all the times.
Without changing the version of GCG, running several times the rake task `bundle exec rake changelog` does not gives always same result. Some times the order is not the good one.